### DISCUSSION TITLE
Will JBrowse2 be able to run on M1 mac?

### DISCUSSION THREAD

**Discussion by Sotdo** [GMOD/jbrowse-components#2827] [General]
Will the JBrowse2 be supported on M1 mac?

**Comment by garrettjstevens** [comment]
It looks like we had a user report they were using JBrowse Desktop on an M1 Mac here: https://github.com/GMOD/jbrowse-components/issues/2818. So maybe it already works? If not, I think our build system would be able to create an M1-specific buid.
